Utilities/Thread.cpp

```diff
@@ -1719,6 +1722,11 @@ void thread_base::initialize(bool(*wait_cb)(const void*))
 #elif !defined(_WIN32)
 	pthread_setname_np(pthread_self(), m_name.get().substr(0, 15).c_str());
 #endif
+
+#ifdef __linux__
+	m_tfd = timerfd_create(CLOCK_MONOTONIC, 0);
+	if (m_tfd == -1) LOG_ERROR(GENERAL, "timer allocation failed, use wait_unlock() only");
```

Comment:
Please use a new line after if, else, for, etc...
Ideally with braces for readability